src/cors.ts

```diff
@@ -0,0 +1,31 @@
+/*******************************************************************************
+
+    Define cors policies
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+ *******************************************************************************/
+import cors from "cors";
+
+// cors policy for agora node
+export const internal_cors_options: cors.CorsOptions = {
+    allowedHeaders: '*',
+    credentials: true,
+    methods: "POST",
+    origin: 'http://127.0.0.1:2826',
+    preflightContinue: false
+};
```

Comment:
It's hard to understand why this is necessary.

Reply:
I don't think it is necessary to expose POST /block_externalized to the outside, so I set a different policy than /GET.

Reply:
I think this will not solve the problem you mentioned.
Also, I think it is different from the current issue.
How about solving other issues?

Reply:
Yes.  Another issue related to this has to be created.  For example, open another port and divid stoa

Reply:
I created an issue related to this. #76 
Current CORS policies can be taken in common.